vocabularies/Common.md

```diff
@@ -166,90 +166,90 @@ Property|Type|Description
 [Note](Common.xml#L416)|String?|Note for error resolution
 [AdditionalNote](Common.xml#L419)|String?|Additional note for error resolution
 
-## <a name="NumericMessageSeverityType"></a>[NumericMessageSeverityType](Common.xml#L471) *([Experimental](Common.md#Experimental))*
+## <a name="NumericMessageSeverityType"></a>[NumericMessageSeverityType](Common.xml#L469) *([Experimental](Common.md#Experimental))*
```

Comment:
Should this one still be experimental if the NumericSeverity is not?

Reply:
Done